ci/r.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+#
+# See https://github.com/r-lib/actions/tree/master/examples#readme for
+# additional example workflows available for the R community.
+
+name: R
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: macOS-latest
```

Comment:
I'm curious that its running `macOS-latest`, could this run on `ubuntu-latest` instead? The cost for a user of running on MacOS is significantly higher than running on Ubuntu, so we'd like to default to that if we can to prevent any surprises.